src/FixedDataTableCellGroup.js

```diff
@@ -180,7 +178,7 @@ class FixedDataTableCellGroupImpl extends React.Component {
 
     return (
       <FixedDataTableCell
-        columnIndex={columnIndex}
+        columnIndex={columnProps.index}
```

Comment:
`columnIndex` previously referred to the local index of the column (i.e, index in relation to the current cell group).
Where as `columnProps.index` refers to the global/actual index of the column (i.e, index in relation to the whole table).